kie-soup-dataset/kie-soup-dataset-elasticsearch/pom.xml

```diff
@@ -1,4 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright ${year} Red Hat, Inc. and/or its affiliates.
```

Comment:
Just noticed the year needs to be set.

Reply:
@mbarkley thanks, I thought that I've fixed those wrong replacements but it seems not